os.hpp

```diff
@@ -1,91 +0,0 @@
-/*
- * os.hpp
- *
- * Baker, Ballard, Jager-Kujawa
- * CSC 341
- * Spring 2016
- */
-
-#include <string>
-#include <queue>
-#include <iostream>
-#include <iomanip>
-#include <cstdlib>
-#include <cstring>
-#include <sstream>
-#include <fstream>
-#include <string>
-#include <cctype>
-#include "machine.hpp"
-  using namespace std;
-
-#ifndef OS
-#define OS
-
-// Define the user struct
-enum userID {sys, u1, u2};
-struct User {
-  userID id;
-  bool running;
-  registers regs;
-};
```

Comment:
This struct should be renamed to something along the lines of PCB, and should contain: 

1. a process ID<sup>1</sup>
2. the registers
3. a pointer to the page table

*<sup>1</sup> For simplicity we can use PID = UID for now (as each user can only have one process at a time for the moment).*